include/pqxx/result.hxx

```diff
@@ -194,6 +194,10 @@ public:
   /// Name of column with this number (throws exception if it doesn't exist)
   [[nodiscard]] char const *column_name(row_size_type number) const &;
 
+  [[nodiscard]] int const column_size(row_size_type number) const;
+
+  [[nodiscard]] int const column_mod(row_size_type number) const;
```

Comment:
Besides the `const` issue (which @KayEss mentioned)...  This also really needs documentation.

Reply:
Since you're addressing a concrete need, not just adding support for more libpq functions for the fun of it, I can help with the documentation — just so long as we don't forget.